extensions/xslt/deployment/src/main/java/org/apache/camel/quarkus/component/xslt/deployment/BuildTimeUriResolver.java

```diff
@@ -48,42 +37,26 @@ public ResolutionResult resolve(String templateUri) throws TransformerException
         final String scheme = ResourceHelper.getScheme(templateUri);
         final String effectiveScheme = scheme == null ? "classpath:" : scheme;
 
-        final String transletName;
-        final Source src;
-        if ("file:".equals(effectiveScheme)) {
-            final String compacted = compact(templateUri, scheme);
-            transletName = toTransletName(compacted);
-            final Path resolvedPath = baseDir.resolve(compacted);
-            try {
-                src = new StreamSource(Files.newInputStream(resolvedPath));
-            } catch (IOException e) {
-                throw new TransformerException("Could not read from file " + templateUri, e);
-            }
-        } else if ("classpath:".equals(effectiveScheme)) {
-            final String compacted = compact(templateUri, scheme);
-            transletName = toTransletName(compacted);
-            final URL url = getClass().getClassLoader().getResource(compacted);
-            try {
-                src = new StreamSource(url.openStream());
-            } catch (IOException e) {
-                throw new TransformerException("Could not read the class path resource " + templateUri, e);
-            }
+        if (!"classpath:".equals(effectiveScheme)) {
```

Comment:
afaik, no scheme is also a synonymous of class path

Reply:
sorry, missed the line above

Reply:
Yes, that's covered by the lines above https://github.com/apache/camel-quarkus/pull/456/files/ecd911af570372f846b242c597f73c9b6ce1c91f#diff-a4d51312ef4951bcb880ba4e9e863d0eR37-R38